uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/controllers/sub-controllers/request-modal-details-leave.controller.spec.js

```diff
@@ -276,6 +276,13 @@ define([
           expect(controller.loading['fromDayTypes']).toBe(false);
           expect(controller.loading['toDayTypes']).toBe(false);
         });
+
+        it('checks for public holiday requests only for the currently selected contact', function () {
+          var args = LeaveRequest.all.calls.mostRecent().args[0];
+
+          expect(args.contact_id).toBeDefined();
```

Comment:
May be better to check if `controller.request.contact_id` is defined rather than `args.contact_id`? It just adds some readability imho.

Reply:
i'm checking if the `LeaveRequest.all()` method had been called with the contact id as the param, not whether the contact id is present in the controller (i need to assume it is)